runtime/src/bank.rs

```diff
@@ -867,6 +874,13 @@ impl Bank {
     pub fn transaction_count(&self) -> u64 {
         self.transaction_count.load(Ordering::Relaxed) as u64
     }
+    pub fn confirmed_transaction_count(&self) -> u64 {
+        if let Some(bank) = self.parents().last() {
```

Comment:
Would this one-liner work instead?
```rust
self.parents().last().unwrap_or(self).transaction_count()
````

Reply:
Not exactly, we've got &self and each parent is &Arc<Bank>